### PR TITLE
Feat/concordex adapter

### PR DESCRIPTION
**Pull request by vladjito**
Add volume and fees data for Concordex DEX on Concordium blockchain

### CODE REVIEW

**Comment by llamatester** [comment]
The concordex-io adapter exports: 
        
 
 _______________________________________
    Dexs for 18/7/2023
    _______________________________________
    
    ------ ERROR ------
    TypeError: Cannot use 'in' operator to search for 'adapter' in undefined
        at exports.default (/home/runner/work/dimension-adapters/dimension-adapters/adapters/utils/getChainsFromDexAdapter.ts:6:18)
        at /home/runner/work/dimension-adapters/dimension-adapters/cli/testAdapter.ts:36:46
        at processTicksAndRejections (node:internal/process/task_queues:95:5)
    Done in 2.90s.

**Comment by llamatester** [comment]
The concordex-io adapter exports: 
        
 
 _______________________________________
    Dexs for 18/7/2023
    _______________________________________
    
    ------ ERROR ------
    TypeError: Cannot use 'in' operator to search for 'adapter' in undefined
        at exports.default (/home/runner/work/dimension-adapters/dimension-adapters/adapters/utils/getChainsFromDexAdapter.ts:6:18)
        at /home/runner/work/dimension-adapters/dimension-adapters/cli/testAdapter.ts:36:46
        at processTicksAndRejections (node:internal/process/task_queues:95:5)
    Done in 2.91s.

File: dexs/concordex-io/index.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+          timestamp: ts,
+          dailyVolume: data.daily_volume,
+          totalVolume: data.total_volume,
+          dailyFees: data.daily_fees,
+          totalFees: data.total_fees,
+        }
+      }
+    }
+  }
+};
```
Comment (dtmkeng) [inline dexs/concordex-io/index.ts:48]: add `export default adapter;` under line 48

**Review by dtmkeng — CHANGES_REQUESTED** [review]

**Comment by llamatester** [comment]
The concordex-io adapter exports: 
        
 
 _______________________________________
    Dexs for 18/7/2023
    _______________________________________
    
    CONCORDIUM 👇
    Backfill start time: 1/7/2023
    NO METHODOLOGY SPECIFIED
    Timestamp: 1689724798
    Daily volume: 20.74
    Total volume: 34564.57
    Daily fees: 0.05
    Total fees: 55.63
    
    
    
    
    Done in 3.57s.

**Pull request merged**